examples/radial_gradient_filters.py

```diff
@@ -0,0 +1,57 @@
+"""
+=======================
+Radial Gradient Filters
+=======================
+
+This example applies both the normalizing radial gradient (`sunkit_image.radial.nrgf`) filter and Fourier
+normalizing radial gradient filter (`sunkit_image.radial.fnrgf`) to a sunpy map.
+
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+
+import sunpy.map
+import sunpy.data.sample
+
+import sunkit_image.radial as radial
+from sunkit_image.utils import equally_spaced_bins
+
+###########################################################################
+
+# Sunpy's sample data contain a number of suitable FITS files for this purpose.
+aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+###########################################################################
+
+# Radial bins are calculated over which the filters will be applied. The distance
```

Comment:
I don't quite understand the first sentence here.

Reply:
Okay I will try to reframe it